pkg/generators/forwarding/fluentd/generators_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package fluentd
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestGenerator(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Generating pipeline to output labels")
+}
```

Comment:
AFAIK, this is obsolete, because a `fluentd_suite_test.go` already exists in this package.

Reply:
yes. i believe we only need one suite file